search/tutorials/buildsearchbar.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: "Docs Search Quickstart"
+description: "A quick guideline to crawl docs and create a functional search component"
```

Comment:
How does something like this sound: `Add Upstash Search to your website in minutes`